apps/i18n/dance/en_us.json

```diff
@@ -7,11 +7,13 @@
   "danceAiFeedbackCustomize": "⚠️ You need to change the effect A.I. generated. You can either regenerate the effects, customize the code A.I. created, or start over with new emojis.",
   "danceAiFeedbackEffectChangeAfterMeasures": "⚠️ Connect the `A.I. generate effect` block to the `after 2 Measures` event.",
   "danceAiFeedbackKeyPress": "⚠️ You pressed a key, but your dancer didn't do a new move.",
-  "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press at least one of the arrow keys while your dance is running.",
+  "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press the right or left arrow key while your dance is running.",
   "danceAiFeedbackMatchDancer": "⚠️ You must have the same dancer selected in the make new dancer block and the dancer do new move block.",
   "danceAiFeedbackNeedNewDancer": "⚠️ You need to make a dancer.",
   "danceAiFeedbackNewMove": "⚠️ You need to add a block to make the dancer do a new move.",
   "danceAiFeedbackNoAiBlock": "⚠️ Add the `A.I. generate effect` block to setup.",
+  "danceAiFeedbackNoNewMove": "⚠️ Your dancer didn't do a new move. Keep trying!",
+  "danceAiFeedbackSequence": "Check the order of your blocks. Make the new dancer before coding the dancer to do a new move.",
```

Comment:
Is it most natural to say "coding the dancer"?  I don't feel like we use that phrase across our product.